locales/fr-FR/messages.po

```diff
@@ -25,6 +25,9 @@ msgstr "Performance moyenne (risque de lenteurs)"
 msgid "Average rendering time"
 msgstr "Temps de rendu moyen"
 
+msgid "Average upload speed"
+msgstr "Bande Passante Montante Moy"
```

Comment:
I did not pay attention, but the AC also mention a dot at the end of the sentence:
`"Bande Passante Montante Moy."`